engine/src/main/java/org/terasology/engine/monitoring/impl/PerformanceMonitorImpl.java

```diff
@@ -113,8 +113,12 @@ public Activity startActivity(String activityName) {
 
         if (!activityStack.isEmpty()) {
             ActivityInfo currentActivity = activityStack.peek();
-            currentActivity.ownTime += newActivity.startTime - ((currentActivity.resumeTime > 0) ? currentActivity.resumeTime : currentActivity.startTime);
-            currentActivity.ownMem += (currentActivity.startMem - newActivity.startMem > 0) ? currentActivity.startMem - newActivity.startMem : 0;
+            currentActivity.ownTime += newActivity.startTime - ((currentActivity.resumeTime > 0)
+                    ? currentActivity.resumeTime
+                    : currentActivity.startTime);
+            currentActivity.ownMem += (currentActivity.startMem - newActivity.startMem > 0)
+                    ? currentActivity.startMem - newActivity.startMem
+                    : 0;
```

Comment:
Is it just me who finds this hard to read and comprehend? Have the conditional outside is already hard, but having it as part of the formula .... :see_no_evil: